code/modules/integrated_electronics/subtypes/arithmetic.dm

```diff
@@ -236,13 +236,13 @@
 // Pi, because why the hell not? //
 /obj/item/integrated_circuit/arithmetic/pi
 	name = "pi constant circuit"
-	desc = "Not recommended for cooking. Outputs '3.14159' when it receives a pulse."
+	desc = "Not recommended for cooking. Outputs '[PI]' when it receives a pulse."
```

Comment:
You can't use string formatting in type definitions, set `desc` in `Initialize()`